internal/cmd/kafka/command_topic_update.go

```diff
@@ -109,6 +107,23 @@ func (c *authenticatedTopicCommand) update(cmd *cobra.Command, args []string) er
 		return nil
 	}
 
+	readOnlyConfigs := types.NewSet()
+	configsValues := make(map[string]string)
+
+	if hasNumPartitionsChanged {
+		updateNumPartitionsInt, err := strconv.Atoi(updateNumPartitions)
```

Comment:
https://pkg.go.dev/strconv#ParseInt with 32 as the last argument is more correct